python/pacemaker/cts/process.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2009-2022 the Pacemaker project contributors
+#
+# The version control history for this file may have further details.
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+
+import psutil
+import sys
+
+from pacemaker.exitstatus import ExitStatus
+
+__all__ = ["killall", "exit_if_proc_running"]
+
+def killall(process_names, terminate=False):
+    """ Kill all instances of every process in a list """
+
+    if not process_names:
+        return
+
```

Comment:
We could also accept a single name as process_names and make a list out of it

Reply:
Added.